### DISCUSSION TITLE
Is there a singular approach to close the security strategy?

### DISCUSSION THREAD

**Discussion by xixici** [langgenius/dify#4178] [General]
When I use python code to execute one agent, it occurs "**operation not permitted**". Is there one method to close the security strategy?

**Comment by patryk20120** [comment]
Some of the services were disabled on the sandbox container due to security reasons, more here https://docs.dify.ai/getting-started/install-self-hosted/install-faq#how-does-the-sandbox-service-ensure-security

**Reply by xixici** [reply]
Well, I get it. But I want to disable the security strategy, let the container be like normal. Is there anyway to solve it?

**Reply by patryk20120** [reply]
The only way I can see is to modify sandbox image and change configuration, which will require custom image generation.

**Reply by ghost** [reply]
Turn this shit off

**Reply by xixici** [reply]
Yes. I have the same idea for this. However, I can't find any wiki for sandbox image. It's difficult to archive.

**Reply by xixici** [reply]
I have gave up trying more python package.
